src/transformers/models/data2vec/modeling_data2vec_audio.py

```diff
@@ -622,6 +622,7 @@ def _flash_attention_forward(
         """
         Calls the forward method of Flash Attention - if the input hidden states contain at least one padding token
         first unpad the input, then computes the attention scores and pad the final attention scores.
+
```

Comment:
These kind of changes are from fix copies, and are not related at all to the PR. But let's leave it here as it's anyway related to code-consistency in the library